src/cascadia/TerminalSettingsEditor/Appearances.cpp

```diff
@@ -41,15 +37,6 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
                 // box, prevent it from ever being changed again.
                 _NotifyChanges(L"UseDesktopBGImage", L"BackgroundImageSettingsVisible");
             }
-            else if (viewModelProperty == L"FontFace")
-            {
-                // notify listener that all font face related values might have changed
-                if (!UsingMonospaceFont())
-                {
-                    _ShowAllFonts = true;
-                }
-                _NotifyChanges(L"ShowAllFonts", L"UsingMonospaceFont");
-            }
```

Comment:
Note: when you move `ShowAllFonts` and `UsingMonospaceFont`, you'll need this code block again.

This is actually a perfect example of why these two properties _need_ to be in the `AppearanceViewModel`. If the JSON is updated manually to a non-monospace font, we _need_ to update `UsingMonospaceFont=false` and `ShowAllFonts=true` (and notify that that may have changed). Otherwise, we might be pointing to a `Font` object that doesn't actually exist in the combo box.

Reply:
The `Appearances` object listens to the view model's property changed event though! So if the view model tells us that the font face changed, we update `UsingMonospaceFont` and `ShowAllFonts` accordingly (effectively this block of code just got moved down to the Appearances object)